test/pthread/test_pthread_barrier.cpp

```diff
@@ -70,9 +70,10 @@ int main(int argc, char **argv)
     int ret = pthread_barrier_init(&barr, NULL, THREADS);
     assert(ret == 0);
 
-    for(int i = 0; i < THREADS; ++i) pthread_create(&thr[i], NULL, &thread_main, (void*)i);
-    for(int i = 0; i < THREADS; ++i)
-    {
+    for (intptr_t i = 0; i < THREADS; ++i) {
+      pthread_create(&thr[i], NULL, &thread_main, (void*)i);
+    }
+    for(int i = 0; i < THREADS; ++i) {
```

Comment:
I'd prefer both `for` have a space after them, but let's keep them consistent at least like before?

Reply:
Done